docs/src/main/asciidoc/telemetry-micrometer-tutorial.adoc

```diff
@@ -77,16 +77,21 @@ Dimensional labels are added for the request uri, the HTTP method
 ----
 # HELP http_server_requests_seconds
 # TYPE http_server_requests_seconds summary
-http_server_requests_seconds_count{method="GET",outcome="SUCCESS",status="200",uri="/example/prime/{number}",} 2.0
-http_server_requests_seconds_sum{method="GET",outcome="SUCCESS",status="200",uri="/example/prime/{number}",} 0.017385896
+http_server_requests_seconds_count{method="GET",outcome="SUCCESS",status="200",uri="/example/prime/{number}"} 2.0
+http_server_requests_seconds_sum{method="GET",outcome="SUCCESS",status="200",uri="/example/prime/{number}"} 0.017385896
 # HELP http_server_requests_seconds_max
 # TYPE http_server_requests_seconds_max gauge
-http_server_requests_seconds_max{method="GET",outcome="SUCCESS",status="200",uri="/example/prime/{number}",} 0.017385896
+http_server_requests_seconds_max{method="GET",outcome="SUCCESS",status="200",uri="/example/prime/{number}"} 0.017385896
 #
 ----
 
 NOTE: Metrics appear lazily, you often won't see any data for your endpoint until it is accessed.
 
+.Exported metrics format
+
+By default, the  metrics are exported using micrometer `application/openmetrics-text`, you can revert to the former
+format by specifying the `Accept` header request to `plain/text`.
```

Comment:
```suggestion
By default, the  metrics are exported using the Micrometer format `application/openmetrics-text`,
you can revert to the former format by specifying the `Accept` request header to `plain/text`.
```

Reply:
Used your suggestions and added a curl exemple

Reply:
Amended accordingly to @ebullient hint